src/helpers/durationToColor.js

```diff
@@ -1,22 +1,22 @@
-import moment from 'moment';
+import moment from "moment";
 
-export default function (value) {
+export default function durationToColor(value) {
   const then = moment().diff(value);
 
   const difference = moment.duration(then);
 
   const duration = moment.duration(difference);
```

Comment:
```js
duration = moment.duration(moment.duration(moment().diff(value)))
```

The double application of `moment.duration` seems like there might be a bug here. Or just a really weird API.

Reply:
I don't know about this; I did not write or review this code. It was probably some linter that encouraged me to use a named exported function or something.

Reply:
to summarize:

- [ ] page title is wrong
- [ ] bright flash on images and videos 
- [ ] console errors
- [ ] last seen not displayed < need example link
- [ ] audio player not working
- [ ] dark mode not defined
- [ ] nsfw classification unavailable
- [ ] metadata panel not working
- [ ] carousel detail page not working

Reply:
update: i think i may have found the document about the last-seen report; and it seems to be fixed, but the description is different:
<img width="902" alt="image" src="https://user-images.githubusercontent.com/625851/181015097-eca8e5fd-e494-418f-93f6-9ee8202a663d.png">

Reply:
@ Fleek build—all above resolved, apart from:

- [ ] dark mode not defined
- [ ] nsfw classification unavailable 

\o/